project/GrasshopperBuild.scala

```diff
@@ -123,5 +125,24 @@ object GrasshopperBuild extends Build {
       )
     )
 
+  lazy val hmdaGeo = ProjectRef(uri("git://github.com/cfpb/hmda-geo.git"), "client")
```

Comment:
I assume this downloads the hmda-geo repo from GH, and builds the "client" sub-project as a dependency?  There doesn't seem to be any way of specifying a version.  Any concern with this approach?  What's the alternative?  Push hmda-geo up to Maven Central?